demos/text_detection_demo/text_detection_demo.hpp

```diff
@@ -43,6 +43,7 @@ static const char input_data_type_message[] = "Required. Input data type: \"imag
                                               "\"video\" (for a saved video), "
                                               "\"webcam\" (for a webcamera device). By default, it is \"image\".";
 static const char utilization_monitors_message[] = "Optional. List of monitors to show initially.";
+static const char decoder_bandwidth_message[] = "Optional. Default bandwidth is 0 it will use CTC Greedy Decoder.";
```

Comment:
2 @FengYen-Chang: I'm sorry, but it requires more detailed explanation, otherwise it won't be clear for users.
```suggestion
static const char decoder_bandwidth_message[] = "Optional. Bandwidth for CTC beam search decoder. Default value is 0, in this case CTC greedy decoder will be used.";
```
I'm sorry, but IMHO it is important.

P.S. Please, change README.md correspondingly.